docs/achtergronddocumentatie/artefacten/8908.md

```diff
@@ -0,0 +1,35 @@
+| `contactpersoonnaam`              | De `contactnaam` van de eventueel als `contactpersoon` geregistreerde   `partij` die betrokken is bij het `klantcontact`                                                                                                                                                      |
+| `binnenlandsCorrespondentieadres` | Ofwel het `correspondentieadres` van de `partij` met wie het klantcontact   werd gevoerd, ofwel (als die niet gekend is of afwijkende gegevens zijn   opgegeven) het `correspondentieadres` van `betrokkene bij klantcontact` in de   rol van `klant` of `vertegenwoordiger`† |
+| `buitenlandsCorrespondentieadres` | Ofwel het `correspondentieadres` van de `partij` met wie het klantcontact   werd gevoerd, ofwel (als die niet gekend is of afwijkende gegevens zijn   opgegeven) het `correspondentieadres` van `betrokkene bij klantcontact` in de   rol van `klant` of `vertegenwoordiger`† |
+| `correspondentiePostadres`        | Ofwel het `correspondentieadres` van de `partij` met wie het klantcontact   werd gevoerd, ofwel (als die niet gekend is of afwijkende gegevens zijn   opgegeven) het `correspondentieadres` van `betrokkene bij klantcontact` in de   rol van `klant` of `vertegenwoordiger`† |
+| `faxnummer`                       | ofwel een `digitaal adres` van de `partij` met wie het klantcontact werd   gevoerd, ofwel (als die niet gekend is of afwijkende gegevens zijn opgegeven)   het `digitaal adres` van `betrokkene bij klantcontact` in de rol van `klant`   of `vertegenwoordiger`‡             |
+| `emailadres`                      | ofwel een `digitaal adres` van de `partij` met wie het klantcontact werd   gevoerd, ofwel (als die niet gekend is of afwijkende gegevens zijn opgegeven)   het `digitaal adres` van `betrokkene bij klantcontact` in de rol van `klant`   of `vertegenwoordiger`‡             |
+| `mijnOverheid`                    | Het `kanaal` dat bij het `klantcontact` werd gebruikt                                                                                                                                                                                                                         |
+| `telefoonnnummer`                 | Ofwel een `digitaal adres` van de `partij` met wie het klantcontact werd   gevoerd, ofwel (als die niet gekend is of afwijkende gegevens zijn opgegeven)   het `digitaal adres` van `betrokkene bij klantcontact` in de rol van `klant`   of `vertegenwoordiger`‡             |
+
+* Onduidelijk is waarom de [specificaties van de Documenten API](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/VNG-Realisatie/gemma-documentregistratiecomponent/1.4.3/src/openapi.yaml#tag/verzendingen/operation/verzending_create) stellen dat dit attribuut verplicht moet worden geregistreerd.
```

Comment:
Gaat dit om het telefoonnummer?  kan ik in de review modus niet echt achterhalen.